fenix/app/src/main/res/values-sw600dp/bools.xml

```diff
@@ -2,6 +2,10 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
     <bool name="tablet">true</bool>
-</resources>
\ No newline at end of file
+
+    <!-- snackbar -->
+    <dimen tools:override="true" name="design_snackbar_min_width">352dp</dimen>
+    <dimen tools:override="true" name="design_snackbar_max_width">536dp</dimen>
```

Comment:
Could you explain how we are getting the values here? I see differing values based on the last comment here https://bugzilla.mozilla.org/show_bug.cgi?id=1807138#c7.

Reply:
Yeah, sure. In that comment the first value is 
`<dimen name="design_snackbar_extra_spacing_horizontal">24dp</dimen>`

So the required width of 328 and of 512 should account for inner spacing of the snackbar. Which leaves us with the 352 and 536 values :)